dbt_project.yml

```diff
@@ -1375,4 +1375,9 @@ models:
     openxswap:
       +schema: openxswap
       optimism:
-        +schema: openxswap_optimism
\ No newline at end of file
+        +schema: openxswap_optimism
+
+    mux_protocol:
+      +schema: mux_protocol
+      optimism:
+        +schema: mux_protocol_optimism
```

Comment:
can you remove any additions to this file and rather put schema directly in model config blocks?